src/meta/MetaServiceUtils.cpp

```diff
@@ -425,27 +375,15 @@ std::string MetaServiceUtils::indexEdgeKey(GraphSpaceID spaceId,
     return key;
 }
 
-std::string MetaServiceUtils::indexTagIndexKey(GraphSpaceID spaceID,
-                                               const std::string& indexName) {
-    EntryType type = EntryType::TAG_INDEX;
+std::string MetaServiceUtils::indexIndexKey(GraphSpaceID spaceID,
+                                            const std::string& indexName) {
     std::string key;
     key.reserve(128);
-    key.append(kIndexTable.data(), kIndexTable.size())
-       .append(reinterpret_cast<const char*>(&type), sizeof(type))
-       .append(reinterpret_cast<const char*>(&spaceID), sizeof(GraphSpaceID))
-       .append(indexName);
-    return key;
-}
-
-std::string MetaServiceUtils::indexEdgeIndexKey(GraphSpaceID spaceID,
-                                                const std::string& indexName) {
-    EntryType type = EntryType::EDGE_INDEX;
-    std::string key;
-    key.reserve(128);
-    key.append(kIndexTable.data(), kIndexTable.size())
-       .append(reinterpret_cast<const char*>(&type), sizeof(type))
-       .append(reinterpret_cast<const char*>(&spaceID), sizeof(GraphSpaceID))
-       .append(indexName);
+    key.append(kIndexTable.data(), kIndexTable.size());
+    EntryType type = EntryType::INDEX;
+    key.append(reinterpret_cast<const char*>(&type), sizeof(type));
```

Comment:
I don't quite get why we put the type before space?

Reply:
I think the index key for each type could save in a range.